admin/settings/advanced.mdx

```diff
@@ -1,17 +1,17 @@
 ---
-title: "Advanced"
+title: "Ingestion Controls"
 ---
 
-The Advanced section is where users can tailor their log tracking infrastructure. The Middleware Host Agent will search for changes in your account configuration and automatically update the configuration YAML file and reload itself.
+The Ingestion Controls section is where users can tailor their log tracking infrastructure. The Middleware Host Agent will search for changes in your account configuration and automatically update the configuration YAML file and reload itself.
```

Comment:
We need to update this intro to include the feature of metrics and traces toggling.